src/script/util/StringUtil.ts

```diff
@@ -161,3 +161,20 @@ export const fixWebsocketString = (originalString: string): string => {
   const decoder = new TextDecoder();
   return decoder.decode(charArray);
 };
+
+const accentsMap: Record<string, string> = {
+  a: 'ä|á|à|ã|â|À|Á|Ã|Â',
+  e: 'ë|é|è|ê|É|È|Ê',
+  i: 'ï|í|ì|î|Í|Ì|Î',
+  o: 'ö|ó|ò|ô|õ|Ó|Ò|Ô|Õ',
+  u: 'ú|ù|û|ü|Ú|Ù|Û|Ü',
+  c: 'ç|Ç',
+  n: 'ñ|Ñ',
+};
+
+/**
+ * @param text The string to replace accents its charachters
+ * @returns new string with replaced accents charachters
+ */
+export const replaceAccents = (text: string) =>
+  Object.keys(accentsMap).reduce((acc, cur) => acc.replace(new RegExp(accentsMap[cur], 'g'), cur), text);
```

Comment:
Do we use the `getSlug` method from `speakingUrl`? 
In this case, let's use the native `custom` property for this ;)
https://github.com/pid/speakingurl#getsluginput-options

Reply:
fixed in https://github.com/wireapp/wire-webapp/pull/14634/commits/ccc090f2a9bbecb23f720d0bb4bcce156a602b15